src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1303,14 +1304,19 @@ public override BoundNode VisitObjectCreationExpression(BoundObjectCreationExpre
                         }
                     }
                 }
+                else if (type.IsNullableType() && arguments.Length == 0)
```

Comment:
This is not correct.  It is possible for a value type to have a declared (in IL) constructor, and if so that is what we use.  The check should include a check that there is no constructor symbol.

Reply:
Can use `IsDefaultValueTypeConstructor()` as above.

---
In reply to: [260569123](https://github.com/dotnet/roslyn/pull/33703#discussion_r260569123) [](ancestors = 260569123)